pkg/eventhandler/eventhandlers.go

```diff
@@ -114,14 +114,25 @@ func (eh *EventHandler) startK8sJob(k8s k8sutils.K8s, action *config.Action, jso
 		// k8s job name max length is 63 characters, with the naming scheme below up to 999 tasks per action are supported
 		jobName := "job-executor-service-job-" + eh.Event.ID()[:28] + "-" + strconv.Itoa(index+1)
 
-		jobErr := k8s.CreateK8sJob(jobName, action, task, eh.EventData, eh.JobSettings, jsonEventData)
+		jobName, err := k8s.CreateK8sJob(jobName, action, task, eh.EventData, eh.JobSettings, jsonEventData)
```

Comment:
As we discussed already, `jobName` as return value would not be necessary here, since it is the same value as passed in?

Reply:
Can we be sure, that it will always be the same? 

If yes, then I will get rid of it :+1:

Reply:
Should be fixed :+1: